configservice/config-service.smithy

```diff
@@ -65,5 +65,8 @@ structure RegistryCredential {
     token: String,
     /// If supplied, username and password will be used for HTTP Basic authentication
     username: String,
-    password: String
+    password: String,
```

Comment:
we should be adding `@sensitive` attribute to password fields.
At the moment it only serves as documentation (it is copied into the rustdocs), and later it might be used to prevent printing to logs